server/itx/server.go

```diff
@@ -277,6 +277,9 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 		log.L().Info("Waiting for server to be ready.", zap.Duration("duration", cfg.API.ReadyDuration))
 		time.Sleep(cfg.API.ReadyDuration)
 	}
+	if err := svr.rootChainService.Blockchain().AddSubscriber(probeSvr); err != nil {
```

Comment:
the subscriber of blockchain receives block after commit, not receiving from network. thus the all api servers will turn off if the entire blockchain network becomes stalled